src/main/java/org/terasology/navgraph/BitMap.java

```diff
@@ -20,12 +20,16 @@
 
 /**
  * @author synopia
+ * Representation of a region or floor
```

Comment:
- what is a floor? is that another class? if so, you can link to it via `{@link ...}` - IntelliJ will help with auto-completion
- what does "representation" mean/what is represented, e.g., what does a bit in this map encode?